stdlib/2and3/filecmp.pyi

```diff
@@ -0,0 +1,49 @@
+# Stubs for filecmp (Python 2/3)
+import sys
+from typing import AnyStr, Callable, Dict, Generic, Iterable, List, Optional, Sequence, Tuple
+
+_SupportsBool = object
```

Comment:
I did this because the default value is 1 in python2 and True on python3 I didn't want people who used 0 to suddenly fail.  If that's not a problem I can switch it to bool or I can make it `Union[int, bool]` if you think it's important.